### PR TITLE
Design changes and all frontend network v3 changes

### PR DESCRIPTION
**Pull request by jchoi330**
Still working on fixing Checkout.test

### CODE REVIEW

File: packages/frontend/tests/AddProductView.test.tsx
Comment (jchoi330) [inline packages/frontend/tests/AddProductView.test.tsx:1]: !!! Ignore all frontend tests for now !!! 
Still refactoring them after the relayClient & stateManager export module

File: packages/client/test/guestCheckout.test.ts [hunk 1]
```diff
@@ -220,63 +277,56 @@ test.skip("single item checkout with a guest", { timeout: 10000 }, async () => {
...
         args: [args],
       })) as bigint;
-      expect(toHex(order.paymentId)).toEqual(toHex(paymentId));
-      // need to wait for the minting of eddies to be done before sending them
-      await mintComplete;
+      expect(toHex(order.paymentId.raw)).toEqual(toHex(paymentId));
+
       // call the pay function
-      guestWallet.writeContract({
+      const hash = await guestWallet.writeContract({
```
Comment (jchoi330) [inline packages/client/test/guestCheckout.test.ts:320]: need to  circle back to this. payTokenPreApproved fails with unknown custom error right now.

**Review by cryptix — CHANGES_REQUESTED** [review]
First pass, i want to review the stateManager and context changes more thoroughly once the tests are working again.

File: packages/frontend/src/app/store/page.tsx [hunk 2]
```diff
@@ -4,86 +4,366 @@
...
+        const file = new File([blob], "file.json");
+        const formData = new FormData();
+        formData.append("file", file);
+        clientWithStateManager!
+          .relayClient!.uploadBlob(formData)
+          .then(({ url }) => {
+            const blockchainClient = new BlockchainClient(shopId!);
+            blockchainClient
+              .setShopMetadataURI(clientWallet!, url)
+              .then()
```
Comment (wanderer) [inline packages/frontend/src/app/store/page.tsx:179]: you can use promise chaining here, instead of nesting the promeses

**Review by wanderer — CHANGES_REQUESTED** [review]

**Comment by cryptix** [comment]
superseeded by devcon-rc1

**Pull request closed without merging**